frontend/src/components/projects/ProjectsDetailedContent.js

```diff
@@ -2,29 +2,40 @@ import React from "react";
 import {bindActionCreators} from "redux";
 import {connect} from "react-redux";
 import {Link, useHistory, useParams} from "react-router-dom";
-import {Descriptions, Typography} from "antd";
+import {Descriptions, Typography, Spin} from "antd";
 const {Title} = Typography;
 
 import AppPageHeader from "../AppPageHeader";
 import PageContent from "../PageContent";
 import TrackingFieldsContent from "../TrackingFieldsContent";
+import ProjectsAssociatedSamples from "./ProjectsAssociatedSamples";
 import {withSample} from "../../utils/withItem";
 import {get} from "../../modules/projects/actions";
 
 const mapStateToProps = state => ({
     projects: state.projects.items,
     isFetching: state.projects.isFetching,
-    projectsByID: state.projects.itemsByID
+    projectsByID: state.projects.itemsByID,
+    samplesByID: state.samples.itemsByID,
 });
 
 const mapDispatchToProps = dispatch =>
     bindActionCreators({ get }, dispatch);
 
-const ProjectsDetailedContent = ({projects, projectsByID, isFetching, get}) => {
+const ProjectsDetailedContent = ({projects, projectsByID, samplesByID, isFetching, get}) => {
     const history = useHistory();
     const {id} = useParams();
     const isLoaded = id in projectsByID;
     const project = projectsByID[id] || {};
+    const isSamplesEmpty = project.samples && project.samples.length === 0;
```

Comment:
Unused variable? 

Trick: I think you are using Pycharm or any JetBrains software right? If there are unused variables they will be grey instead of coloured 

Ex: 
![image](https://user-images.githubusercontent.com/1617741/133157379-275de172-effd-4f12-8b12-929ca092cfd6.png)

(same applies for unused packages) ex:
![image](https://user-images.githubusercontent.com/1617741/133157426-a047f9c6-4231-470f-a678-770ef2b45a19.png)